src/mango-avocado-salad.md

```diff
@@ -0,0 +1,22 @@
+# Mango and avocado salad
+
+- ⏲️  Prep time: 5 min
+- 🍳 Cook time: 1 min (optional)
+- 🍽️ Servings: 2 * n (number of fruits)
+
+## Ingredients
+- (n) Mangoes
+- (n) Avocados
+- Seeds (optional) (sesame, pumpkin...??)
+
+## Directions
+1. Peel and chop fruit into chunks.
+2. Add to bowl and mix.
+3. Optionally toast seeds and serve on top.
+
+## Contribution
+- [locua](https://github.com/locua)
+
+;tags: salad fruit asian
+
+
```

Comment:
```suggestion
;tags: salad fruit asian
```

Reply:
Don't know why the diff looks so weird, but basically just meant to remove the last 2 empty lines in the file